user_tools/src/spark_rapids_tools/configuration/common.py

```diff
@@ -59,29 +54,21 @@ class DependencyVerification(BaseModel):
 
 
 class RuntimeDependency(BaseModel):
-    """The runtime dependency required by the tools Jar cmd. All elements are downloaded and added
-    to the classPath."""
-    name: str = Field(description='The name of the dependency')
+    """Holds information about a runtime dependency required by the tools' java cmd."""
+    name: str = Field(
+        description='The name of the dependency.',
+        examples=['Spark-3.5.0', 'AWS Java SDK'])
     uri: Union[AnyUrl, FilePath] = Field(
-        description='The FileURI of the dependency. It can be a local file or a remote file',
-        examples=['file:///path/to/file.jar',
+        description='The location of the dependency file. It can be a URL to a remote web/storage or a file path.',
+        examples=['file:///path/to/file.tgz',
                   'https://mvn-url/24.08.1/rapids-4-spark-tools_2.12-24.08.1.jar',
                   'gs://bucket-name/path/to/file.jar'])
     dependency_type: RuntimeDependencyType = Field(
         default_factory=lambda: RuntimeDependencyType(dep_type=DependencyType.get_default()),
-        description='The type of the dependency and how to find the lib files after decompression.',
+        description='Specifies the dependency type to determine how the item is processed. '
+                    'For example, jar files are appended to the java classpath while archive files '
+                    'such as spark are extracted first before adding subdirectory _/jars/* to the classpath.',
         validation_alias=AliasChoices('dependency_type', 'dependencyType'))
     verification: DependencyVerification = Field(
         default=None,
-        description='The verification information of the dependency.',
-        examples=[
```

Comment:
nit: are the examples removed on purpose?

Reply:
Yes, because the examples were not in the correct level.
This caused some confusion in understanding the file.

Reply:
thanks @amahussein!